localMatchApp/headers/events.h

```diff
@@ -0,0 +1,74 @@
+#ifndef EVENTS_H
+#define EVENTS_H
+
+#include <QWidget>
+
+/*!
+ * The class of Event is created to
+ * be able to develop the functions
+ * to manage the variables of the event
+ */
+class Events
```

Comment:
Change this class name also to describe it better. Note that "Events" is a plural word, which means that an object of Events type contains multiple events, I think that was not what you intended.